### DISCUSSION TITLE
Kan hver pakke sin url lede til pakken i stedet for prosjekt-rot?

### DISCUSSION THREAD

**Discussion by joms** [fremtind/jokul#2372] [Annet]
Er det mulig å peke URLen til hver pakke mot pakkens readme, eller komponentside i stedet for prosjekt-rot?

![image](https://user-images.githubusercontent.com/537060/136794739-3fe4107b-9c53-4d6f-aa7c-2f57658fcf9f.png)

**Comment by wkillerud** [comment]
Flytter denne over til Discussions 👮

**Comment by wkillerud** [comment]
Er åpen for å gjøre det :) Hvordan gjøres det typisk i andre monorepos?

**Reply by joms** [reply]
Dunno tbh, det kan jeg prøve å grave i. Kjente bare at det ble et umiddelbart behov når jeg skulle oppgradere pakker 😅

**Comment by piofinn** [comment]
Synes det virker som en god idé! For min del gir det kanskje mest mening med lenke til portalsiden til komponenten? Det finnes jo også en repository-lenke som vil peke til koden på github, og vi vil vel uansett at portalsiden skal være den riktigste kilden til info..

![Skjermbilde 2021-10-12 kl  12 11 57](https://user-images.githubusercontent.com/25739615/136937127-6e86649e-ba91-4b5a-a867-aacd89f17a7b.png)
